sig-security-assessments/Documentation/recipe-book.md

```diff
@@ -0,0 +1,49 @@
+# Recipe Book 👩‍🍳
+🎉Yay! You have decided to do a Self-assessment and make your corner of Kubernetes more secure! We are so happy 
+to have you here. Below is an outline of the journey to get to a successful Self-assessment.
+
+## Preparation
+Set yourself up for success – build your team and get organized
+1.	Figure out who can do this Self-assessment with you. You will need a security expert and a counterpart who owns the 
+project and is an expert in it. If you are requesting a Security Self-assessment you can reach out to the team in
+[#sig-security](https://kubernetes.slack.com/messages/sig-security) to see if there is someone interested in helping to
+champion the assessment.
+1.	Make sure that you and your counterpart each have a back-up person assigned who attends all the meetings and is 
+involved in the assessment. Two heads are better than one, and redundancy means that if someone can’t make a meeting 
+for whatever reason, that you can still make progress.
+1.	To get further participation, advertise in SIG meetings and channels for the project, as well as SIG-Security. Aim for a minimum of 4 
+people (a project expert, a security expert, and a backup for each), and a maximum of 8. This ensures that there is 
+not overwhelming burden on any one individual but allows for a smooth decision-making process.
+1.	Create a Google doc for meeting notes using 
```

Comment:
question: At what point do we want Self-assessment project teams to open a request for a slack channel?

Reply:
Hm - I think once there is a solid group of people who are interested in participating in the self assessment that a slack channel becomes necessary for coordination and publicity/socializing that a self-assessment is underway. /2p